src/components/Tabs/Tabs.tsx

```diff
@@ -217,16 +184,9 @@ export const Tabs = ({
     scrollableRight && styles['tabs--scrollable-right'],
   );
 
-  const childrenWithProps = React.Children.map(getTabs(), (child, i) => {
-    // Checking isValidElement is the safe way and avoids a typescript
-    // error too.
-    if (React.isValidElement(child)) {
-      return React.cloneElement<Props>(child, {
-        id: tabPanelIds[i],
-        'aria-labelledby': tabIds[i],
-      });
-    }
-    return child;
```

Comment:
No need to clone every child, since we're only rendering one of them (the active tab panel).